### DISCUSSION TITLE
Alternative theme for the docs:

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [MartinBernstorff/swift-python-cookiecutter#122] [General]
The sphinx book theme - looks pretty solid as well.
https://sphinx-book-theme.readthedocs.io/en/stable/

Actually quite like the entire framework o MyST stuff - you could even [write a paper](https://curvenote.com/blog/writing-a-scientific-paper-faster-myst-markdown) in it. Pretty sexy I must say

**Comment by MartinBernstorff** [comment]
I'm completely agnostic here, so if you want it you're more than welcome! :-)

**Comment by KennethEnevoldsen** [comment]
I also most left it here for me to take a look at it later, but i generally like the look that we have now. The book theme is probably better for bigger projects though.

**Comment by MartinBernstorff** [comment]
Maybe so! Is it OK that I move it to a discussion, then? My intuition is that issues should be for, well, issues (bugs, missing features, insufficient performance etc.) ;-)
